docs/application/native/guides/text-input/voice-control.md

```diff
@@ -70,6 +72,8 @@ The following figure illustrates the voice control life-cycle states.
 
 <a name="elementary"></a>
 ## Voice Control with Elementary Components
+[!NOTE]
+All Voice Control Elementary APIs are deprecated since Tizen 10.0 and will be removed after two releases without any alternatives.
```

Comment:
```suggestion
> [!NOTE]
> All Voice Control Elementary APIs are deprecated since Tizen 10.0 and will be removed after two releases without any alternatives.
```